.clabot

```diff
@@ -48,7 +48,9 @@
     "christopherdro",
     "patrickworkman",
     "PJijin",
-    "adeelhasan"
+    "adeelhasan",
+    "KosmosKey",
```

Comment:
Added myself as an contributor. @KosmosKey did the commit, but pull request was raised from @easyrun42 👍